genai-perf/genai_perf/telemetry_data/telemetry_data_collector.py

```diff
@@ -46,6 +46,18 @@ def __init__(
         self._stop_event = Event()
         self._thread: Optional[Thread] = None
 
+    def is_url_reachable(self) -> bool:
+        TIMEOUT_SECONDS = 5
```

Comment:
Should this be all caps? I think you only do this with module-level constants, not function variables.

Reply:
yep, sorry i moved it to the function but didn't make it lowercase.